src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -162,6 +163,10 @@ public void addTag(Tag t) throws UniqueTagList.DuplicateTagException {
         tags.add(t);
     }
 
+    public void removeTag(Tag t) throws TagNotFoundException {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.
In this case since `removeTag` throws an exception, it would not be a trivial method. 
A header comment will help the programmer that uses this method understand that there is an exception to be handled.